features/support/capybara.rb

```diff
@@ -1,117 +1,48 @@
 # frozen_string_literal: true
 
 require 'selenium/webdriver'
-require 'capybara/minitest'
-include Capybara::Minitest::Assertions
-
-# in case firefox is needed!
-#Capybara.register_driver :selenium do |app|
-#  Capybara::Selenium::Driver.new(app, :browser => :firefox)
-#end
 
 DEFAULT_JS_DRIVER = :headless_chrome
-# in case firefox is needed!
-# DEFAULT_JS_DRIVER = :headless_firefox
-
-Capybara.default_driver = :rack_test
-Capybara.javascript_driver = DEFAULT_JS_DRIVER
-Capybara.default_selector    = :css
-Capybara.disable_animation = true
 
-# Capybara 3 changes the default server to Puma. It can be reverted to the previous default of WEBRick by specifying:
-Capybara.server = :webrick
-
-# see http://www.elabs.se/blog/60-introducing-capybara-2-1
 Capybara.configure do |config|
   config.default_driver = :rack_test
-  config.javascript_driver = DEFAULT_JS_DRIVER
+  config.default_selector = :css
```

Comment:
These first three lines are actually the defaults, so they could be removed potentially. I decided to keep them, because I think it's clearer if they are explicit and not hidden within Capybara's code (or docs).

But if anybody thinks they are redundant, we can drop them.

Reply:
Did you finally remove them?